aldryn_jobs/models.py

```diff
@@ -241,10 +239,7 @@ def get_absolute_url(self, language=None):
         category_slug = self.category.safe_translation_getter(
             'slug', language_code=language
         )
-        if self.app_config_id:
-            namespace = self.app_config.namespace
-        else:
-            namespace = 'aldryn_jobs'
+        namespace = getattr(self.category.app_config, "namespace", "aldryn_jobs")
```

Comment:
the reason I bring it up is, if you do this here then you'd have to do it in other places where we rely on app_config.